addons/website_hr_recruitment/static/src/scss/website_hr_recruitment.scss

```diff
@@ -1,24 +1,29 @@
 .o_website_hr_recruitment_jobs_list {
-    #jobs_grid_left {
-        ul.flex-column > li.nav-item > a {
-            padding: .5rem 1rem;
-            border-radius: 0.125rem;
-        }
-    }
-    .card {
-        padding: 15px;
-        box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.17);
-
-        &:hover {
-            transition: box-shadow 0.3s cubic-bezier(0.55, 0, 0.1, 1);
-            box-shadow: 0 6px 10px 0 rgba(0, 0, 0, 0.1), 0 2px 2px 0 rgba(0, 0, 0, 0.05);
-        }
+    #jobs_grid {
         .o_job_infos {
             .fa {
-                color: theme-color('primary');
-                margin-right: 8px;
+                color: theme-color('secondary');
+                margin-right: .5rem;
```

Comment:
In this case, this change should be kept, especially since we are using a widget in the 't-field' as a 't-option', which invokes a fa icon by default, but it doesn't allow to add any class customisation, unless the widget is being modified to do so.

https://github.com/odoo-dev/odoo/blob/32238cfd27ab23379a795a26b2b1f8770085034f/addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml#L258